### PR TITLE
Fixes example snippets in CharacterProperties.swift

### PR DESCRIPTION
**Pull request by valeriyvan**
Fixes example snippets in CharacterProperties.swift

### CODE REVIEW

File: stdlib/public/core/CharacterProperties.swift [hunk 1]
```diff
@@ -32,7 +32,7 @@ extension Character {
   ///
   ///     let chars: [Character] = ["a", " ", "™"]
   ///     for ch in chars {
-  ///         print(ch, "-->", ch.properties.numericValue)
+  ///         print(ch, "-->", ch.asciiValue)
   ///     }
   ///     // a --> 97
```
Comment (CodaFi) [inline stdlib/public/core/CharacterProperties.swift:37]: The accessor change also results in an `Optional` value being printed.

File: stdlib/public/core/CharacterProperties.swift [hunk 2]
```diff
@@ -125,7 +125,7 @@ extension Character {
   ///
   ///     let chars: [Character] = ["4", "④", "万", "a"]
   ///     for ch in chars {
-  ///         print(ch, "-->", ch.properties.numericValue)
+  ///         print(ch, "-->", ch.wholeNumberValue)
```
Comment (CodaFi) [inline stdlib/public/core/CharacterProperties.swift:128]: Same here

File: stdlib/public/core/CharacterProperties.swift (hunk 1 above)
Comment (valeriyvan) [reply stdlib/public/core/CharacterProperties.swift:37]: `.properties.numericValue` used to be optional as well.

Comment (valeriyvan) [reply stdlib/public/core/CharacterProperties.swift:37]: The style of showing outputs of all snippets in this file doesn't show exactly what is printed.
Do you think it should be changed to following?
```
/// Prints:
/// a --> Optional(97)
///   --> Optional(32)
/// ™ --> nil
```

Comment (CodaFi) [reply stdlib/public/core/CharacterProperties.swift:37]: Yes

Comment (valeriyvan) [reply stdlib/public/core/CharacterProperties.swift]: Fixed

File: stdlib/public/core/CharacterProperties.swift (hunk 2 above)
Comment (valeriyvan) [reply stdlib/public/core/CharacterProperties.swift:129]: fixed

**Comment by CodaFi** [comment]
@swift-ci smoke test

**Comment by CodaFi** [comment]
⛵

**Pull request merged**